pkg/scheduler/factory/factory.go

```diff
@@ -911,6 +893,24 @@ func (c *configFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 	}, nil
 }
 
+// getBinderFunc returns a func which returns an extender that supports bind or a default binder based on the given pod.
+func getBinderFunc(client clientset.Interface, extenders []algorithm.SchedulerExtender) func(pod *v1.Pod) Binder {
```

Comment:
Should we move those help functions into separated file, e.g. `factor/util.go`. Maybe a separated PR.

Reply:
agreed. my hope is to move it outside of the factory package. when phase :panda_face: completes, it should be easier to accomplish that.